<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgAbout</class>
 <widget class="QDialog" name="DlgAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>About subutai tray</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources/resources.qrc">
    <normaloff>:/hub/about.png</normaloff>:/hub/about.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_3" rowstretch="10,0,0">
     <item row="2" column="0">
      <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,0">
       <item row="0" column="1">
        <widget class="QProgressBar" name="pb_initialization_progress">
         <property name="maximum">
          <number>7</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btn_recheck">
         <property name="text">
          <string>Recheck</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_about_init">
         <property name="text">
          <string>Collecting data:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2" rowstretch="2,0,0,0,0,0,0,0,0,0,0" columnstretch="1,0,0,0">
       <item row="6" column="3">
        <widget class="QPushButton" name="btn_subutai_e2e">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Subutai E2E</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lbl_oracle_virtualbox_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lbl_subutai_e2e_val">
         <property name="text">
          <string>undefined</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="lbl_p2p_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="lbl_x2go_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QProgressBar" name="pb_x2go">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QPushButton" name="btn_rh_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update RH</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btn_tray_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update Control Center</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="lbl_vagrant_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QProgressBar" name="pb_p2p">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lbl_rhm_version">
         <property name="text">
          <string>Management version:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QProgressBar" name="pb_vagrant">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="3">
        <widget class="QPushButton" name="btn_oracle_virtualbox_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update VirtualBox</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_vagrant_version">
         <property name="text">
          <string>Vagrant</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QProgressBar" name="pb_e2e">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btn_p2p_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update P2P</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lbl_rh_version">
         <property name="text">
          <string>RH version:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="lbl_rh_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lbl_x2go_version">
         <property name="text">
          <string>X2Go-Client</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QProgressBar" name="pb_rhm">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QProgressBar" name="pb_tray">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QProgressBar" name="pb_oracle_virtualbox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="5" column="3">
        <widget class="QPushButton" name="btn_rhm_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Update Management</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btn_vagrant_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Vagrant</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lbl_subutai_e2e">
         <property name="text">
          <string>Subutai E2E</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QProgressBar" name="pb_rh">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="btn_x2go_update">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update X2Go-Client</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_p2p_version">
         <property name="text">
          <string>P2P version:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbl_tray_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="lbl_rhm_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lbl_oracle_virtualbox_version">
         <property name="text">
          <string>Oracle VirtualBox</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lbl_chrome_version">
         <property name="text">
          <string>Google Chrome</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="lbl_chrome_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QProgressBar" name="pb_chrome">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lbl_firefox">
         <property name="text">
          <string>Mozilla Firefox</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="lbl_firefox_version_val">
         <property name="text">
          <string>undefined</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QProgressBar" name="pb_firefox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="8" column="3">
        <widget class="QPushButton" name="btn_chrome">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Chrome</string>
         </property>
        </widget>
       </item>
       <item row="10" column="3">
        <widget class="QPushButton" name="btn_firefox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Update Firefox</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_tray_version">
         <property name="text">
          <string>Control Center version:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
  <include location="../../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
